domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -157,15 +159,19 @@
 
   private static final Logger log = LoggerFactory.getLogger(PostgresClient.class);
 
+
   private static int embeddedPort            = -1;
 
+  private static long explainQueryThreshold = EXPLAIN_QUERY_THRESHOLD_DEFAULT;
```

Comment:
Can you add some javadoc about the unit (seconds?) and whether there is a magic value (-1?) to disable it.